src/test/javascript/spec/component/shared/metis/postings-thread/postings-thread.component.spec.ts

```diff
@@ -74,120 +40,16 @@ describe('PostingThreadComponent', () => {
                 fixture = TestBed.createComponent(PostingThreadComponent);
                 component = fixture.componentInstance;
                 metisService = TestBed.inject(MetisService);
```

Comment:
You can remove the metis service here I assume

Reply:
Thanks, unused variable is removed.